src/app/modules/edit-profile/edit-profile.component.html

```diff
@@ -0,0 +1,325 @@
+<div class="row">
+  <div class="col-xs-12 col-md-6 col-md-offset-3">
+    <form #authForm="ngForm" (ngSubmit)="onSubmit(authForm)">
+      <div class="form-group">
+        <label for="email">EMAIL ADDRESS</label>
+        <input
+          type="email"
+          id="email"
+          class="form-control"
+          [(ngModel)]="this.user.email"
+          name="email"
+          required
+          email
+          #email="ngModel"
+          (click)="resetMessages()"
+        />
+        <span class="help-block" *ngIf="!email.valid && email.touched">Please enter valid email!</span>
+        <span class="help-block" *ngIf="this.duplicateEmail">Email already in use</span>
+      </div>
+      <div class="form-group">
+        <label for="password">SET NEW PASSWORD</label>
+        <input
+          type="password"
+          id="password"
+          class="form-control"
+          [(ngModel)]="this.user.password"
+          name="password"
+          #password="ngModel"
+          validatePassword
+          (click)="resetMessages()"
+        />
+        <span class="help-block" *ngIf="(!password.valid && password.touched)">
+          Password should be at least 8 characters long and contain one of each characters: lowercase character, uppercase character, number and special character
+        </span>
+      </div>
+      <div class="form-group">
+        <label for="location">YOUR LOCATION</label>
+        <select
+          id="location"
+          class="form-control"
+          [(ngModel)]="this.user.location"
+          name="location"
+          required
+          #location="ngModel"
+          (click)="resetMessages()"
+        >
+          <option value="AF">Afghanistan</option>
```

Comment:
https://stackoverflow.com/questions/33181936/how-to-use-select-option-ngfor-on-an-array-of-objects-in-angular2
Create array of values and use ngFor